public/app/dashboard/editProfile.html

```diff
@@ -1,5 +1,3 @@
-<a ui-sref="dashboard" class="col-back"><i class="fa fa-angle-left"></i> Back to Impact Studio</a>
-
```

Comment:
Wise feedback from QA suggested the removal above and `h1` change below. (The cancel button is obvious 'Back to profile' link.)